docs/static/releasenotes.asciidoc

```diff
@@ -13,6 +14,46 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-1-3]]
+=== Logstash 6.1.2 Release Notes
+
+* Fix bug where with terminating input plugins in-memory queue might not be drained. This could happen in some situations with inputs like the stdin input or the Elasticsearch input. This could result in some messages not being processed.
+
```

Comment:
May want to call out Windows: correctly handle paths with spaces #8931

Reply:
Great call, not sure how I missed that.